boxsdk/object/upload_session.py

```diff
@@ -20,10 +21,20 @@ class UploadSession(BaseObject):
     _item_type = 'upload_session'
     _parent_item_type = 'file'
 
-    def get_url(self, *args: Any) -> str:
+    def __init__(
+            self, session: Session, object_id: str, response_object: dict = None, use_upload_session_urls: bool = True
+    ):
+        super().__init__(session, object_id, response_object)
+        self._use_upload_session_urls = use_upload_session_urls
+
+    def get_url(self, *args: Any, url_key: str = None) -> str:
```

Comment:
In case of existing customer using custom `API. UPLOAD_URL` they will be use the new URL after this change?

Reply:
Yes, unless they set use_upload_session_urls=False

Reply:
So in case you are aware of this situation, this PR is good to go :D